core/templates/css/oppia-material.css

```diff
@@ -0,0 +1,2644 @@
+/* Colors for the ripple elements.*/
+/* stylelint-disable-next-line material/theme-mixin-api */
+/* stylelint-disable-next-line material/theme-mixin-api */
+/* stylelint-disable material/no-prefixes */
+/* stylelint-enable */
+/* stylelint-disable material/no-prefixes */
+/* stylelint-enable */
```

Comment:
This part is quite confusing, also do we expect this file to be modified later? IF not i might make sense to write something like: "DO NOT MODIFY" here. And maybe the instructions on how this was generated can also be included here.